### ISSUE TITLE
Keyboard Keys not retained with new version installs

### ISSUE THREAD

**Issue by GeneralUser-NonDev** [clsid2/mpc-hc#2109]
First off, sorry, this is also a feature request too, unless it's already here somewhere and I just can't find it. If I knew how to do this kind of stuff I would (among other things I'd like to fix not related to MCP), but unfortunately as my s/n suggests I don't. Thank you any and all people for your effort put into this project.

I've noticed then when updating/upgrading to new version my settings (Key mappings, display bar size, etc) don't persist through the install (or file copy). I'd like a way to save/export/import the settings (Including but not limited to keyboard mappings) that would persist through updates or even be able to be uploaded into new installs whether on the same system or another.

I don't recall where but I'd seen an option to backup something to a .ini file, But have yet to figure out how to load it into another install (Upgrade or new); I'd tried it quite a while ago (way older release) to no avail.

**Comment by clsid2** [comment]
Settings are preserved unless you uninstall.

You can backup here:
Options > Miscellaneous

**Comment by GeneralUser-NonDev** [comment]
Humm, is the installer different from copy/pasting the zip folder over the older install (what I've been doing), maybe that's what I was doing wrong, I just recall having to re-do my key mappings upon changing versions; Maybe it was "Downgrading" that was my issue when a feature I wanted was removed and I went backwards maybe that did the erase. 

How would I then import these saved settings into another instance (either another computer or a re-install) I don't see an import button or a file of that name in the folder structure of the current install to copy that backup file over. a backup has no use if it can't be restored.
Note: My "Export keys" is greyed out, I only have "Export" (and "Reset") is the keys saved under that one too, or is that just general settings? is the "Reset" also the import, and just not clear? I haven't tried that on my other computer yet (Fresh Install).
